crates/transaction-pool/src/validate/eth.rs

```diff
@@ -627,6 +631,8 @@ impl EthTransactionValidatorBuilder {
             blob_store: Box::new(blob_store),
             kzg_settings,
             local_transactions_config,
+            max_tx_input_bytes: DEFAULT_MAX_TX_INPUT_BYTES, /* TODO: Best way to bring poolconfig
+                                                             * to the scope? */
```

Comment:
wdym here exactly?

Reply:
i should use `max_tx_input_bytes` from cli here, i put question in pr's comment.